Actions/BuildALDoc/action.yaml

```diff
@@ -0,0 +1,35 @@
+name: Build Reference Documentation
```

Comment:
Not sure how much it matters, but the action name typically matches the folder name of the action.